config/samples/cosmos_v1_cosmosfullnode_full.yaml

```diff
@@ -99,6 +99,14 @@ spec:
   service:
     # Creates N LoadBalancer services and exposes those public IPs in p2p.
     maxP2PExternalAddresses: 1
+    # Configures all p2p services that need external addresses.
+    p2pTemplate:
+      metadata:
+        labels:
+          extra: labels
+        annotations:
+          extra: annotations
+      type: NodePort
```

Comment:
@agouin Let me know if there is more we need to configure. We can also configure external traffic policy which is moot point because service always points to 1 pod.